sdk/src/Core/Amazon.Runtime/Credentials/Internal/AwsV4aAuthScheme.cs

```diff
@@ -0,0 +1,29 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using Smithy.Identity.Abstractions;
+
+namespace Amazon.Runtime.Credentials.Internal
```

Comment:
Do we also have to add HttpBearer? `smithy.api#HTTPBearerAuth`? I see in the description that you're trying to get Sigv4 services to work first, so does that mean this will be added in an upcoming PR?

Reply:
Yes, we do need it but I'll add it separately (bearer token uses a separate implementation - it resolves credentials at request time but that needs to be adapted to the SRA).